tasks/Bartender.tex

```diff
@@ -0,0 +1,29 @@
+\section{Bartender [Party Host]}
+The home owner will teach the robot to make a drink which guests at the party really like!
+
+\subsection{Focus}
+Learning from demonstration.
+
+\subsection{Main Goal}
+The home owner will teach the robot using demonstrations how to make a drink. The robot is then to repeat the task. A designated team member will show the home owner how to perform the demonstration. These drinks need to be \textit{good}, meaning that the robot must pour each mixer in for the right amount of time to get the right amount of mixer in.
+
+\noindent\textbf{Reward:} 2500pts (400 points per ingredient).
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item Screwing a lid onto a screw-top blender (500 points)
```

Comment:
Blenders are expensive, we are not sure teams will try this at all, and you need to add the blender to the objects section under General rules.